src/js/components/search/visualizations/geo/map/MapBox.jsx

```diff
@@ -134,13 +215,67 @@ export default class MapBox extends React.Component {
         this.map.removeSource(layerId);
     }
 
+    handleWindowResize() {
+        // determine if the width changed
+        const windowWidth = window.innerWidth;
+        if (this.state.windowWidth !== windowWidth) {
+            // width changed, update the visualization width
+            this.setState({
+                windowWidth
+            }, () => {
+                if (this.map) {
+                    this.resizeMap();
+                }
+                else {
+                    this.mountMap();
+                }
+            });
+        }
+    }
+
     render() {
+        let hideClass = '';
+        if (this.state.showNavigationButtons === false) {
+            hideClass = ' hide';
+        }
+
         return (
             <div
                 className="mapbox-item"
                 ref={(div) => {
                     this.mapDiv = div;
-                }} />
+                }}>
+                <div className={`map-buttons ${hideClass}`}>
+                    <div className="first-row">
+                        <button
```

Comment:
Is there a reason for accessing the buttons via refs instead of setting up `onMouseDown`/`onMouseUp`/`onTouchStart`/`onTouchEnd` props?